transcendence-app/src/shared/dtos/pagination.query.dto.ts

```diff
@@ -9,7 +9,7 @@ export class PaginationQueryDto {
   @IsOptional()
   @IsInt()
   @Min(0)
-  @Max(MAX_ENTRIES_PER_PAGE)
+  @Max(2147483648)
```

Comment:
We somehow need to allow string 'ALL' for limit query parameter.

Reply:
Maybe I don't need this, since I only need the chatroommember - authuser intesection, not all of the chatroom members